appengine-java8/cloudsql/pom.xml

```diff
@@ -79,7 +79,7 @@
     <dependency>
       <groupId>com.google.cloud.sql</groupId>
       <artifactId>mysql-socket-factory</artifactId> <!-- mysql-socket-factory-connector-j-6 if using 6.x.x -->
```

Comment:
what does that comment above mean ? Java 6 ? can you make that more explicit or remove if it is no longer required